config-model/src/main/java/com/yahoo/vespa/model/admin/LogserverContainerCluster.java

```diff
@@ -21,7 +19,7 @@ public LogserverContainerCluster(AbstractConfigProducer<?> parent, String name,
 
         addDefaultHandlersWithVip();
         addLogHandler();
-        setJvmGCOptions(deployState.getProperties().jvmGCOptions(Optional.of(ClusterSpec.Type.admin)));
+        setJvmGCOptions(deployState.featureFlags().jvmOmitStackTraceInFastThrowOption(ClusterSpec.Type.admin));
```

Comment:
Intentional to remove jvmGCOptions ? Not append instead ?

Reply:
Not intentional, thanks, will fix